e2e/support/helpers/e2e-dashboard-visualizer-helpers.ts

```diff
@@ -112,6 +112,47 @@ export function verticalWell() {
   return cy.findByTestId("vertical-well");
 }
 
+export function assertWellItems(items: {
+  horizontal?: string[];
+  vertical?: string[];
+  pieMetric?: string[];
+  pieDimensions?: string[];
+}) {
+  const { horizontal, vertical, pieMetric, pieDimensions } = items;
+
+  if (horizontal) {
+    horizontalWell().within(() => {
+      horizontal.forEach((item) => {
+        cy.findByText(item).should("exist");
+      });
+    });
+  }
```

Comment:
nit: it could be helpful to also assert the number of items in a well to ensure we're not showing anything unexpected